src/r_exasol/connection/protocol/meta_info_reader.cpp

```diff
@@ -26,8 +26,8 @@ std::pair<std::string, uint16_t>  exa::metaInfoReader::read(exa::Socket& socket)
 
     if ((r = socket.recv((void*)&(proxyAnswer), sizeof(proxyAnswer))) != sizeof(proxyAnswer)) {
         std::stringstream stringStream;
-        stringStream << "Failed to receive proxy header from " << socket.getConnectionInfo().first << ":" <<
```

Comment:
I see, we don't have the connection info anymore

Reply:
Actually this error message here was the only place where the connection info was being used. And with the new SSLSocket we would have to copy this info. So I found it better to remove the connection info object, and add the needed parameters here.